### ISSUE TITLE
no-duplicate-imports rule triggered for `import type` statements

### ISSUE THREAD

**Issue by elliottsj** [babel/babel-eslint#293]
eslint [recently introduced a 'no-duplicate-imports' rule](https://github.com/eslint/eslint/blob/25b376c9752cd0c1c48c62784ad42d3d6e3ebcdf/docs/rules/no-duplicate-imports.md) which is triggered if there are two or more `import` statements from the same module.

It seems that this rule is also triggered for `import type` Flow statements, e.g. this results in an error:

``` js
import { foo } from 'someModule';
import type { Bar } from 'someModule';
```

afaik, there's no way to combine these two statements into one `import` statement. Would this be an issue with babel-eslint since it provides the Flow syntax support?

**Comment by danez** [comment]
The rule is not differentiating between type and regular imports and sadly babel-eslint can't do anything about that as far as I can see. Probably this rule is a good candidate for adding to https://github.com/babel/eslint-plugin-babel and adjusting it to differ between these two import types.

**Comment by nicolo-ribaudo** [comment]
Closing in favor of https://github.com/babel/eslint-plugin-babel/issues/59
